scene/2d/animated_sprite_2d.cpp

```diff
@@ -154,10 +161,7 @@ void AnimatedSprite2D::_validate_property(PropertyInfo &p_property) const {
 void AnimatedSprite2D::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_INTERNAL_PROCESS: {
-			if (frames.is_null()) {
-				return;
-			}
-			if (!frames->has_animation(animation)) {
+			if (frames.is_null() || !frames->has_animation(animation)) {
 				return;
 			}
 			if (frame < 0) {
```

Comment:
This check may not be necessary. If it is to be checked, the last_frame should also be checked. However actuality, the frame value is changed immediately after here, and the setter prevents the entry of an incorrect value.

Reply:
A bunch of these early returns are technically impossible to reach, I believe. I didn't want to change them in this PR to avoid any obscure breaking behaviour.

Reply:
Perhaps that was to be deleted in #64155. I would like to remove it now.

Reply:
Can do